specs/ServerCertificate.md

```diff
@@ -0,0 +1,299 @@
+void EnableCustomServerCertificateError()
+{
+  if (!_isServerCertificateError)
+  {
+    webView.CoreWebView2.ReceivingServerCertificateError += WebView_ReceivingServerCertificateError;
+  }
+  else
+  {
+    webView.CoreWebView2.ReceivingServerCertificateError -= WebView_ReceivingServerCertificateError;
+  }
+  _isServerCertificateError = !_isServerCertificateError;
+
+  MessageBox.Show(this,
+    _isServerCertificateError ? "Custom server certificate error has been enabled" : "Custom server certificate error has been disabled",
+    "Custom server certificate error");
+}
+
+void WebView_ReceivingServerCertificateError(object sender, CoreWebView2ReceivingServerCertificateErrorEventArgs e)
+{
+  CoreWebView2Certificate certificate = e.ServerCertificate;
+
+  // Continues with the navigation to a server with a TLS certificate if
+  // the error status is of type `COREWEBVIEW2_WEB_ERROR_STATUS_CERTIFICATE_IS_INVALID`
+  // and trusted by the host app.
+  if (e.ErrorStatus == CoreWebView2WebErrorStatus.CertificateIsInvalid &&
+                            ValidateServerCertificate(certificate))
+  {
+    e.Handled = true;
+  }
+  else
+  {
+    // Cancel the request for other TLS certificate error types or if untrusted by the host app.
+    e.Cancel = true;
+  }
+}
+
+// Function to validate the server certificate for untrusted root or self-signed certificate.
+bool ValidateServerCertificate(CoreWebView2Certificate certificate)
+{
+    // You may want to validate certificates in different ways depending on your app and scenario.
+    // One way might be the following:
+    // First, get the list of host app trusted certificates and its thumbprint.
+
+    // Then get the last chain element using `ICoreWebView2Certificate::get_PemEncodedIssuerCertificateChain`
+    // that contains the raw data of the untrusted root CA/self-signed certificate. Get the untrusted
+    // root CA/self signed certificate thumbprint from the raw certificate data and
+    // validate the thumbprint against the host app trusted certificate list.
+
+    // Finally, return true if it exists in the host app's certificate trusted list, or otherwise return false.
+    return true;
+}
+
+// This example clears the TLS decision in response to proceeding with TLS certificate errors.
+async void ClearServerCertificateErrorOverrideCache()
+{
+  bool isSuccessful = await webView.CoreWebView2.ClearServerCertificateErrorOverrideCacheAsync();
+  string message = (isSuccessful) ?
+                        "Clear server certificate error override cache succeeded." :
+                        "Clear server certificate error override cache failed.";
+  MessageBox.Show(this, "message", "Clear server certificate error override cache");
+}
+```
+
+# API Details
+
+## Win32 C++
+``` cpp
+[uuid(4B7FF0D2-8203-48B0-ACBF-ED9CFF82567A), object, pointer_default(unique)]
+interface ICoreWebView2_11 : ICoreWebView2_10 {
+  /// Add an event handler for the ServerCertificateError event.
```

Comment:
Update comment to match final event name.

Reply:
Will update